src/style.css

```
@@ -0,0 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(-n+1),
```

update like this. For cases below as well
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
```